community-operators/seldon-operator/seldonoperator.v0.1.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,312 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: seldonoperator.v0.1.2
+  namespace: seldon-operator-system
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: "Logging & Tracing"
+    description: The Seldon operator for management, monitoring and operations of machine learning systems through the Seldon Engine. Once installed, the Seldon Operator provides multiple functions which facilitate the productisation, monitoring and maintenance of machine learning systems at scale.
+    containerImage: seldonio/seldon-core-operator:0.2.8-SNAPSHOT
+    createdAt: 2019-05-21T15:00:00Z
+    repository: https://github.com/SeldonIO/seldon-operator
+    alm-examples: '[{"apiVersion": "machinelearning.seldon.io/v1alpha2","kind": "SeldonDeployment","metadata": {"labels": {"app": "seldon"},"name": "seldon-model"},"spec": {"name": "test-deployment","oauth_key": "oauth-key","oauth_secret": "oauth-secret","predictors": [{"componentSpecs": [{"spec": {"containers": [{"image": "seldonio/mock_classifier:1.0","imagePullPolicy": "IfNotPresent","name": "classifier","resources": {"requests": {"memory": "1Mi"}}}],"terminationGracePeriodSeconds": 1}}],"graph": {"children": [],"name": "classifier","endpoint": {"type" : "REST"},"type": "MODEL"},"name": "example","replicas": 1,"labels": {"version" : "v1"}}]}}]'
+    certified: "false"
+    support: Clive Cox
+spec:
+  displayName: Seldon Operator
+  minKubeVersion: 1.13.0
+  description: |
+    The Seldon operator enables for native operation of production machine learning workloads, including monitoring and operations of language-agnostic models with the benefits of real-time metrics and log analysis.
+       
+    ## Installation
+    In order to install the Seldon Operator you need to make sure you create a secret in the namespace. You can do so by creating a file called `seldon-operator-secret.yaml` with the following contents:
+    ```
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: seldon-operator-webhook-server-secret
+      namespace: default
+
+    ```
+    ## Deploy a model
```

Comment:
This is not necessary. The example is already mentioned under alm-examples metadata.

Reply:
@axsaucedo  also remove 

> Thank you @aneeshkp - I've added the changes requested. Should I modify the upstream-operator to align as well in this PR? Or create a new PR for this?
> 
> In regards to the final suggestion, let me have a chat with the team to see what can be done.

For Upstream changes create new PR, also you can remove the "deployment model details" from the description .  The examples are already mentioned under alm-examples metadata.
you can preview the CSV here https://operatorhub.io/preview